src/planner/macos/mod.rs

```diff
@@ -364,6 +368,40 @@ fn check_not_running_in_rosetta() -> Result<(), PlannerError> {
     Ok(())
 }
 
+async fn check_suis() -> Result<(), PlannerError> {
+    let policies: profiles::Policies = match profiles::load().await {
+        Ok(pol) => pol,
+        Err(e) => {
+            tracing::info!(
+                "Error loading profile data: {:?}, skipping SystemUIServer checks.",
+                e
+            );
+            return Ok(());
+        },
+    };
+
+    let blocks: Vec<_> = profile_queries::blocks_internal_mounting(&policies)
+        .into_iter()
+        .map(|tpopt| tpopt.display())
```

Comment:
What is `tpopt`? Can we use better a name, please?

Reply:
this part of p'tawww